src/dscanner/main.d

```diff
@@ -237,7 +239,12 @@ else
 		if (skipTests)
 			config.enabled2SkipTests;
 		if (report)
-			generateReport(expandArgs(args), config, cache, moduleCache);
+		{
+			if (reportFormat == "sonarQubeGenericIssueData")
```

Comment:
maybe this should be made a switch-case so a warning message can be logged on default (with fallthrough)

switch-case also implies that there may be more formats to be added in the future.

Reply:
fixed